app/routes/search.tsx

```diff
@@ -17,11 +17,25 @@ export const loader: LoaderFunction = async ({ request }) => {
     });
   }
 
-  const events = await db.event.findMany({ include: { group: true } });
+  // for pagination in the future, use the "skip" property in conjunction with the "take" property
+  const events = await db.event.findMany({
+    include: { group: true },
+    where: {
+      OR: [
+        { location: { contains: query } },
```

Comment:
no need for query or col to have lower, because
Prisma SQL is case insensitive by default:
https://www.prisma.io/docs/concepts/components/prisma-client/case-sensitivity#mysql-provider